opentelemetry/src/common.rs

```diff
@@ -462,6 +462,28 @@ impl InstrumentationScope {
             attributes: None,
         }
     }
+
+    /// Instrumentation library name.
```

Comment:
nit: Use the word "return" in the documentation for all the getters.

```suggestion
    /// Returns the instrumentation library name.
```